recipes/pyfftw/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set version = "0.10.1" %}
+
+package:
+    name: pyfftw
+    version: {{ version }}
+
+source:
+    fn: pyFFTW-{{ version }}.tar.gz
+    url: https://pypi.python.org/packages/source/p/pyFFTW/pyFFTW-{{ version }}.tar.gz
+    md5: dab57de9d23f3b333115265914f3d240
+
+build:
+  number: 0
+  skip: True  # [win]
+
+requirements:
+    build:
+        - cython
+        - python
+        - numpy x.x
+        - fftw
+
+    run:
+        - python
+        - numpy x.x
+        - fftw
+
+test:
+    imports:
+        - pyfftw
+        - pyfftw.builders
+        - pyfftw.interfaces
+
+about:
+    home: http://hgomersall.github.com/pyFFTW/
+    license: BSD or GPL 2
```

Comment:
I know this is a bit weird, but the code itself is BSD. Technically if you pay the licensing fee for FFTW then you can get a permissive license (I believe it is MIT) instead of GPL 2, then this would technically be BSD. However, parts that link and are used with the GPL 2 version of FFTW (not paying the licensing fee) will obviously then be GPL'd.